src/style.css

```diff
@@ -2,0 +1,173 @@
+/* #region Reset Css */
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+font,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font-weight: inherit;
+  font-style: inherit;
+  font-size: 100%;
+  font-family: inherit;
+  vertical-align: baseline;
+}
+
+/* #endregion */
+
+/* #region Stars 0 */
+
+.stars--0 {
+  display: flex;
+}
+
+/* #endregion */
+
+/* #region Stars 1 */
+.stars--1 {
+  display: flex;
+}
+
+.stars--1__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: left;
+  margin: 0 2px;
+}
```

Comment:
The classes `.stars--1__star`, `.stars--2__star`, `.stars--3__star`, `.stars--4__star`, `.stars--5__star` all have the same properties. You can simplify your code by creating a single `.stars__star` class with these properties and applying it to all your star elements.